data-prepper-plugins/opensearch/src/main/java/org/opensearch/dataprepper/plugins/source/opensearch/configuration/AwsAuthenticationConfiguration.java

```diff
@@ -32,6 +32,9 @@ public class AwsAuthenticationConfiguration {
     @JsonProperty("serverless")
     private Boolean serverless = false;
 
+    @JsonProperty("serverless_options")
+    private ServerlessOptions serverless_options;
```

Comment:
For the Java variable we should name this `serverlessOptions`. The property name as represented in `@JsonPreoperty` is good and should have the underscore.